src/praisonai/praisonai/auto.py

```
@@ -1140,7 +1190,7 @@ def __init__(self, topic: str = "Research and write about AI",
         self.framework = framework
         self.single_agent = single_agent
     
-    def recommend_pattern_llm(self, topic: str = None) -> PatternRecommendation:
+    def recommend_pattern_llm(self, topic: str = None) -> Any:
```

Ruff (0.15.18)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)